### ISSUE TITLE
Add rate limiting for job submission

### ISSUE THREAD

**Issue by bilalshaikh42** [reproducible-biomedical-modeling/CRBM-Viz#1715]
@mpw6 @moraru What are your thoughts on reasonable limits/maximum limits for a given IP or user

**Comment by jonrkarr** [comment]
Skip this for now. This will be a good problem to have to deal with.

**Comment by bilalshaikh42** [comment]
The primary concern was the risk of a script/test  or bug in the code sending requests rapidly. Unlikely, but could cause issues with jobs on the hpc

**Comment by moraru** [comment]
I agree with Jonathan that this can wait.  We have some decent monitoring to safeguard against runaway job submission.  I’ll check with Mike to make sure it applies to the runbiosim dedicated resources.

No need to implement it in the API for now.  Later on what we need to do is to support soft and hard quotas.

**Comment by bilalshaikh42** [comment]
Great, thank you!
